src/components/Header/AvatarMenu.tsx

```diff
@@ -72,7 +72,12 @@ export const AvatarMenu = ({
               }
             />
           </AvatarMenuButton>
-          <Menu.List role="menu" marginTop="0.375rem" {...menuListProps}>
+          <Menu.List
+            role="menu"
+            marginTop="0.375rem"
+            zIndex={10}
```

Comment:
nit - shall we use a constant for this? I was thinking having a seperate file for this where we can define stuff like 
Z-INDEX__FOR_NOTIFICATIONS = 10
Z-INDEX_FOR_ANNOUNCEMENTS = 50 
^ was thinking this would be neater as we know exactly which `level` each components are - rather than us guessing that 10 is sufficiently enough! wdyt?

Reply:
Chakra has default hints - eg: `dropdown`, we should use those

Reply:
Good point! I missed out on Chakra's one. Updated in 870d1ef6.

I am opting to use Chakra's one instead of defining our own set of constants as this is already done for the SiteViewHeader/SiteEditHeader, only changing the values to match the ones defined for those.